### PR TITLE
EAPI 5 cleanup

### PR DESCRIPTION
**Pull request by 00-matt**
Bump some packages to EAPI 7 and remove some unused inherits.

### CODE REVIEW

File: app-misc/cdctl/cdctl-0.16.ebuild [hunk 1]
```diff
@@ -1,29 +1,22 @@
...
 
 LICENSE="free-noncomm"
 SLOT="0"
 KEYWORDS="amd64 ppc ppc64 x86"
-IUSE=""
-
-DEPEND=""
 
-S="${WORKDIR}/${PN}"
+PATCHES=("${FILESDIR}"/${PN}-0.16-Makefile.in.patch)
```
Comment (thesamesam) [inline app-misc/cdctl/cdctl-0.16.ebuild:17]:
```suggestion
PATCHES=( "${FILESDIR}"/${PN}-0.16-Makefile.in.patch )
```

File: app-metrics/prom2json/prom2json-0.1.0_p20170523.ebuild [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-inherit user golang-build golang-vcs-snapshot
```
Comment (thesamesam) [inline app-metrics/prom2json/prom2json-0.1.0_p20170523.ebuild:5]: New line after EAPI for m-n and sort inherit if it doesn't break phase export order (if stuff still gets called as needed). You can skip this if not m-n as it's style though

File: net-analyzer/nettop/nettop-0.2.3-r2.ebuild [hunk 3]
```diff
@@ -19,12 +17,10 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	epatch \
-		"${FILESDIR}"/${P}-gcc411.patch \
-		"${FILESDIR}"/${P}-offbyone.patch
-	tc-export CC
```
Comment (SoapGentoo) [inline net-analyzer/nettop/nettop-0.2.3-r2.ebuild:26]: you sure this isnt needed anymore? did you check with `USE="-native-symlinks?"`?

File: dev-vcs/git-crypt/git-crypt-0.6.0.ebuild [hunk 4]
```diff
@@ -15,6 +15,6 @@ RDEPEND="dev-vcs/git"
 DEPEND="${RDEPEND}"
```
Comment (SoapGentoo) [inline dev-vcs/git-crypt/git-crypt-0.6.0.ebuild:15]: block structure for `LICENSE` and friends is wrong

File: dev-libs/cxxtools/cxxtools-2.2.1.ebuild [hunk 5]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
 DESCRIPTION="Collection of general purpose C++-classes"
```
Comment (SoapGentoo) [inline dev-libs/cxxtools/cxxtools-2.2.1.ebuild:6]: `src_install` seems pointless here

File: app-misc/figlet/figlet-2.2.5-r1.ebuild [hunk 6]
```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit eutils bash-completion-r1 toolchain-funcs
+EAPI=7
+inherit bash-completion-r1 toolchain-funcs
```
Comment (SoapGentoo) [inline app-misc/figlet/figlet-2.2.5-r1.ebuild:5]: empty line before `inherit`

File: app-misc/crunch/crunch-3.6.ebuild [hunk 7]
```diff
@@ -13,14 +11,6 @@ LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~mips ~x86"
 
-src_prepare() {
-	epatch_user
-}
-
-src_compile() {
-	emake CC="$(tc-getCC)"
```
Comment (SoapGentoo) [inline app-misc/crunch/crunch-3.6.ebuild:21]: sure?

Comment (00-matt) [reply app-misc/crunch/crunch-3.6.ebuild:21]: Nope, I misunderstood why it was needed. Thanks for catching :) got it explained on IRC. 
Will revert.

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**